service/controller/resource/monitoring/prometheus/test/case-1-awsconfig.golden

```diff
@@ -32,6 +32,11 @@ spec:
       certFile: /etc/prometheus/secrets/cluster-certificates/crt
       keyFile: /etc/prometheus/secrets/cluster-certificates/key
   arbitraryFSAccessThroughSMs: {}
+  containers:
```

Comment:
This fields allow to override any container field, see https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#prometheusspec